app/assets/stylesheets/common.scss

```diff
@@ -5,6 +5,8 @@
 @import "modules/popup_panels";
 @import "modules/utility";
 
+@import "https://fonts.googleapis.com/css2?family=Source+Sans+3:ital,wght@0,200..900;1,200..900&display=swap";
```

Comment:
Do we use all these weights? Might be able to save a few kB if it's only 300,400,700 which are more common

Reply:
I was using the same URL as in the component-library, so that the user will have a cached version.

Reply:
Interesting. I guess if we get people to cache them all across our sites, it'll matter less